hack/images/dcos/Makefile

```diff
@@ -27,7 +27,28 @@ KUBE_DNS_TEMPLATES_SOURCES = $(KUBE_DNS_TEMPLATES:%=../dns/%)
 
 S6_BINS ?= $(S6_OBJ) $(EXECLINE_OBJ)
 
-.PHONY:	clean build push s6 $(S6_BINS) deps etcd $(ETCD_BINS) $(KUBE_DNS_TEMPLATES)
+#
+# normally you'll be generating dev images via:
+#	make release 
+#
+# and promoting (re-tagging them with a release version) them via:
+#	make promote
+#
+# HOWEVER, to generate a fresh release image and push it up (skip the -dev tag cycle):
+#	make release FROM=~/bin RELEASE=1
```

Comment:
will do

On Thu, May 14, 2015 at 7:40 PM, Connor Doyle <notifications@github.com>
wrote:

> In hack/images/dcos/Makefile
> <https://github.com/mesosphere/kubernetes-mesos/pull/278#discussion_r30374505>
> :
>
> > @@ -27,7 +27,28 @@ KUBE_DNS_TEMPLATES_SOURCES = $(KUBE_DNS_TEMPLATES:%=../dns/%)
> >
> >  S6_BINS ?= $(S6_OBJ) $(EXECLINE_OBJ)
> >
> > -.PHONY:	clean build push s6 $(S6_BINS) deps etcd $(ETCD_BINS) $(KUBE_DNS_TEMPLATES)
> > +#
> > +# normally you'll be generating dev images via:
> > +#	make release
> > +#
> > +# and promoting (re-tagging them with a release version) them via:
> > +#	make promote
> > +#
> > +# HOWEVER, to generate a fresh release image and push it up (skip the -dev tag cycle):
> > +#	make release FROM=~/bin RELEASE=1
>
> This would be useful to note in the README as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/kubernetes-mesos/pull/278/files#r30374505>.
>